packages/grid/src/features/standard-table/config/StandardTableConfig.ts

```diff
@@ -8,6 +8,23 @@ export interface RowExpansionArgs {
   onRequestCollapse?: () => void;
 }
 
+export interface RowHeaderOptions {
```

Comment:
Should we do nested objects?
We avoid them in props obviously, but we never use them in configs either. Is it a good idea to start now?

Reply:
Is it better to have name prefixes on these instead? so instead of the options: headerRowIsSticky, headerRowTop, headerRowZIndex.